README.md

```diff
@@ -465,6 +481,7 @@ A list of **Free** and **Open Source Software** ***(FOSS)*** for **Android** –
 * [**Phonograph Plus**](https://github.com/chr56/Phonograph_Plus) <sup>**[[F-Droid](https://f-droid.org/app/player.phonograph.plus)]**</sup>
 * [**raag**](https://github.com/raag-music/raag) <sup>**[[F-Droid](https://f-droid.org/app/in.amfoss.raag)]**</sup>
 * [**Simple Music Player**](https://github.com/SimpleMobileTools/Simple-Music-Player) <sup>**[[F-Droid](https://f-droid.org/app/com.simplemobiletools.musicplayer)]**</sup>
+* [**SimpMusic**](https://github.com/maxrave-dev/SimpMusic) <sup>**[[IzzyOnDroid](https://apt.izzysoft.de/fdroid/index/apk/com.maxrave.simpmusic)]**</sup>
```

Comment:
It's on F-Droid already: https://f-droid.org/app/com.maxrave.simpmusic

According to it's readme it's still beta though. Maybe we put it on hold, but no strong opinion here – no objection.

Reply:
I guess they're really close for stable release hence published on F-Droid ⸮ Updated F-Droid link.